### ISSUE TITLE
The getter '' isn't defined for the type 'Colors'.

### ISSUE THREAD

**Issue by ibytao** [dart-lang/sdk#50391]
<img width="877" alt="Screenshot 2022-11-05 at 10 37 12" src="https://user-images.githubusercontent.com/1676997/200097431-180bff7d-c538-40e0-a740-30f151857b74.png">

<img width="2518" alt="Screenshot 2022-11-05 at 10 35 15" src="https://user-images.githubusercontent.com/1676997/200097381-a56e0c23-baba-411a-b5dd-f0f8a364cc80.png">

**Comment by asashour** [comment]
Please submit text instead of images, as it is easier to read, and is possibly searchable.

A simplified version:
```dart
class A {}
void f() {
  A.;
}
```

There are two errors, one of them is:

>The getter '' isn't defined for the type 'A'

**Comment by scheglov** [comment]
Yes, suppressing subsequent diagnostic for synthetic identifiers (or tokens) is a good idea.
